add_user.php

```diff
@@ -53,26 +53,27 @@
 										$insertquery = "INSERT INTO Users( User_PK, Password ) VALUES( '$UserID', '$PW' )";
 									?>
 									<center>
+										<img style="width: 100%; max-width: 200px;" src="images/placeholder.png" alt="Profile graphic">
 										<h1>ADD USER</h1>
-										<h3><form method = "post" id="add_user.php">
-											<label for = 'login'>Username*</label><br />
-											<!-- input box for the user to enter their username -->
-											<input type = "text" name = "username" placeholder="Enter Username"/><br />
-											<label for = 'login'>Password*</label><br />
-											<!-- input box for the user to enter their password -->
-											<input type = "password" name = "password" id="myInput" placeholder="Enter Password"/><br />
-											<!-- An element to toggle between password visibility -->
-											<input type="checkbox" onclick="myFunction()">Show Password<br /><br />
-											<!-- Button for the user to press once they are ready to login -->
-											<input type = "submit" value = "Create Account"/><br />
+										<h3><form method = "post" id="add_user">
+										<label for="username">Username*</label>
+										<!-- input box for the user to enter their username -->
+										<input type = "text" name = "username"  autocomplete="off" placeholder="Enter Username"/><br /><br />
+										<label for = 'password'>Password*</label>
+										<!-- input box for the user to enter their password -->
+										<input type = "password" name = "password" id="myInput" placeholder="Enter Password"/><br />
+										<!-- An element to toggle between password visibility -->
+										<input type="checkbox" onclick="myFunction()"><p style="display: inline-block; color: #527ECC;">Show Password</p><br />
+										<!-- Button for the user to press once they are ready to login -->
+										<input type = "submit" value = "Create Account"/><br />
 										</form></h3>
-
+										
 										<?php
 											//lets the admin know the status of the action; whether or not the user was successfully added or they have to try again.
 											if (mysqli_query($conn,$insertquery)) {
-												echo "<h3>New User Successfully Registered</h3>";
+												echo "<p style='color: green; font-weight: 700;'>New User Successfully Registered</p>";
```

Comment:
#added colour to notifications and error messages to signify whether  action was successful (green) or not (red)